flake.nix

```diff
@@ -97,7 +100,8 @@
           }
         ) config.rust;
 
-        nixosConfigurations.default =
+        nixosConfigurations = lib.mkIf hasAnyWebServer {
+          default =
```

Comment:
We already have a condition further down, which checks whether the webserver is enabled. That is now redundant, I think. We should only have one condition, right?

Reply:
Never mind.